src/cdef.rs

```diff
@@ -181,9 +181,9 @@ fn padding<BD: BitDepth>(
     let stride = src.pixel_stride::<BD>();
 
     // Fill extended input buffer.
-    let mut x_start = 2 - 2;
+    let mut x_start = 0;
```

Comment:
It was `2 - 2` for a reason.  This shouldn't be simplified.  Can you leave it as is?